### ISSUE TITLE
Berlin changeset won't update with new payload

### ISSUE THREAD

**Issue by stevevance** [transitland/transitland#177]
I edited the [Berlin/Brandenburg changeset](https://transit.land/dispatcher/changesets/3139) offline because it has so many operators. And now it won't upload. Several operators were combined, but I took care to cross-reference Onestop IDs between the `operator` objects and the `feed` object. 

The new changeset is attached: [3139_berlin.txt](https://github.com/transitland/transitland/files/587010/3139_berlin.txt)

**Comment by irees** [comment]
It contains some repeated values in `includesOperators`

Example:

```
				}, {
					"gtfsAgencyId": "NEV---",
					"operatorOnestopId": "o-u33-nebbetriebsgesellschaftmbh"
				}, {
					"gtfsAgencyId": "NEV---",
					"operatorOnestopId": "o-u33-nebbetriebsgesellschaftmbh"
				}, {
```

I removed them and updated the changeset:
https://transit.land/dispatcher/changesets/3139

**Comment by stevevance** [comment]
@irees Did you find that automatically or manually?

**Comment by irees** [comment]
Ok - one more fix needed. An operator is referenced by more than one agency_id. Known issue, see https://github.com/transitland/transitland/issues/81
Can be resolved after the fix for that issue is tested.

**Comment by stevevance** [comment]
This was resolved, but Berlin/Brandenburg is too large a feed to import.

**Comment by aardila** [comment]
@stevevance does this mean that we shouldn't expect Berlin data to be available ever? [valhalla/valhalla#12](https://github.com/valhalla/valhalla/issues/12)
